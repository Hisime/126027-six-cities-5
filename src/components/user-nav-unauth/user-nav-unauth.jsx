import React, {Fragment} from "react";

const UserNavUnauth = () => {
  return (
    <Fragment>
      <div className="header__avatar-wrapper user__avatar-wrapper"/>
      <span className="header__login">Sign in</span>
    </Fragment>
  );
};

export {UserNavUnauth};
