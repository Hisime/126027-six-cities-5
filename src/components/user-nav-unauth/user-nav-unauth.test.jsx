import React from "react";
import renderer from "react-test-renderer";
import {UserNavUnauth} from "./user-nav-unauth";

it(`Should UserNavUnauth render correctly`, () => {
  const tree = renderer
    .create(<UserNavUnauth/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
