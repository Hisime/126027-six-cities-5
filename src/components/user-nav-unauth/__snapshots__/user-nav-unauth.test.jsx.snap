// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should UserNavUnauth render correctly 1`] = `
Array [
  <div
    className="header__avatar-wrapper user__avatar-wrapper"
  />,
  <span
    className="header__login"
  >
    Sign in
  </span>,
]
`;
